rtt/TaskContext.cpp

```diff
@@ -349,6 +349,8 @@ namespace RTT
             new_act = new SequentialActivity();
 #elif defined(ORO_ACT_DEFAULT_ACTIVITY)
             new_act = new Activity();
+#else
+            return false;
```

Comment:
Not sure if I understand what you mean.
If `new_act == 0` but `ORO_ACT_DEFAULT_SEQUENTIAL` or  `ORO_ACT_DEFAULT_ACTIVITY` are defined  a new default activity of the proper type is created and the old one (if existing) is stopped.
So wouldn't be inconsistent to use this method to clear the activity of the TaskContext only if both definitions are not set?